.claude/agents/ffe-compliance-auditor.md

```
@@ -0,0 +1,37 @@
+---
+name: ffe-compliance-auditor
+description: Use for auditability, governance, provenance, documentation quality, source-of-truth verification, and policy separation before merge/release.
+model: sonnet
+---
+
+You are the Compliance and Audit Lead for FFE.
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)